Observer/OrderSaveAfter.php

```diff
@@ -57,22 +59,26 @@ function execute(Observer $ob) {
 							? '{"error": "Magento webhook failed to encode order, please look at order ' . $order_id . ' on website to see the details."}'
 							: $encoded_str;
 						$res = $this->post($payload_str, $o->getStore());
+					} catch (OLE $e) {
+						$comment[] = 'Stock2Shop Webhook exception: ' . $e->getMessage();
 					} catch (\Exception $e) {
 						$this->exception_msg = 'Stock2Shop Webhook exception: ' . $e->getMessage();
 						$this->logger->error($this->exception_msg);
 					}
 
 					// Set errors as webhook response, if any
 					$errors = $this->getErrors();
-					$res = !empty($errors) ? implode(', ', $errors) : $res;
-
-					$comment = [
-						"The Stock2Shop's webhook is notified."
-						,"The order's status: «<b>{$status}</b>»."
-						,"The order's state: «<b>{$state}</b>»."
-						,sprintf("The webhook's response: «<b>%s</b>».", mb_substr($res, 0, 25000))
-					];
-					if (!empty($errors)) {
+
+					$comment[] = "The Stock2Shop's webhook is notified.";
+					$comment[] = "The order's status: «<b>{$status}</b>».";
+					$comment[] = "The order's state: «<b>{$state}</b>».";
+
+					if (!empty($errors) || !empty($res)) {
+						$res       = !empty($errors) ? implode(', ', $errors) : $res;
+						$comment[] = sprintf("The webhook's response: «<b>%s</b>».", mb_substr($res, 0, 25000));
+					}
+
+					if (!empty($errors) && !empty($payload)) {
```

Comment:
There's no need to check `!empty($errors)` right?

Reply:
[1e297cc](https://github.com/stock2shop/magento2_module_webhook/pull/12/commits/1e297cc630087ecb6958a67f8e590d0bd4e198c6)